src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -79,17 +79,17 @@ class PeerActor[R <: HandshakeResult](
         processHandshakerNextMessage(initHandshaker, rlpxConnection.copy(uriOpt = Some(uri)), numRetries)
 
       case RLPxConnectionHandler.ConnectionFailed =>
-        log.warning("Failed to establish RLPx connection")
+        log.debug("Failed to establish RLPx connection")
```

Comment:
After discussion, `ConnectionFailed` is a regular common occurrence and probably not a `warning` level log and if we turn OFF logging we never see any future `info` messages. We could go either way with this but I think getting our logging into a consistent state  will be less of a "dark art" if we push the common occurrences into the `debug` category.